recipes-debian/gtk+3/gtk+3_debian.bb

```diff
@@ -0,0 +1,26 @@
+# base recipe: meta/recipes-gnome/gtk+/gtk+3_3.24.5.bb
+# base branch: warrior
+# base commit: 86be661fd6ae90abc5108c7cba56bdcefa812eab
+
+require ${COREBASE}/meta/recipes-gnome/gtk+/gtk+3.inc
+
+BPN = "gtk+3.0"
+inherit debian-package
+require recipes-debian/sources/${BPN}.inc
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/gtk+-${PV}"
+
+FILESPATH_append = ":${COREBASE}/meta/recipes-gnome/gtk+/gtk+3"
+SRC_URI += "\
+    file://0001-Hardcoded-libtool.patch \
+    file://0002-Do-not-try-to-initialize-GL-without-libGL.patch \
+    file://0003-Add-disable-opengl-configure-option.patch \
+    file://link_fribidi.patch \
+"
+
+S = "${WORKDIR}/gtk+-${PV}"
```

Comment:
The debian-package class sets S to DEBIAN_UNPACK_DIR, thereby you don't have to set S most of times.

Reply:
I have removed the line.